packages/api/cms-api/src/generator/generate-crud.ts

```diff
@@ -280,6 +340,16 @@ function generateArgsDto({ generatorOptions, metadata }: { generatorOptions: Cru
                 : ""
         }
 
+        ${
+            hasStatusFilter
+                ? `
+        @Field(() => ${statusFilterClassName}, { defaultValue: ${statusFilterDefaultValue} })
+        @IsEnum(${statusFilterClassName})
+        status: ${statusFilterClassName};
```

Comment:
It's weird that both the status filter for the dialogs and the data grid have status filter in their name. Should we call the former differently? For instance `view: ProductView`?

Reply:
Hm, ProductView is too unspecific, you can't guess what this does.